website_wow/README.rst

```diff
@@ -0,0 +1,62 @@
+.. image:: https://img.shields.io/badge/license-GPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/gpl-3.0-standalone.html
+   :alt: License: GPL-3
+
+===========
+Website Wow
+===========
+
+Provides animations for your Odoo website using WowJS, which is powered by `animate.css`.
+
+This is created as a replacement to Odoo's `website_animate` module, which uses proprietary
+animations build by Odoo for Odoo. Animate.css is a more widely used framework that is more
+efficient, better maintained, and contains significantly more animations.
+
+.. image:: static/description/screenshot.png?raw=true
+   :alt: Data Sliders
+
+Usage
+=====
+
```

Comment:
(list of things to do - I honestly just kind of started writing this app, so I wasn't really sure where I was going with it until I did)